src/ArticleTheme.vala

```diff
@@ -0,0 +1,87 @@
+using Gee;
+
+public class  FeedReader.ArticleTheme {
+    static Array<HashMap> themes = new Array<HashMap> ();
+
+    public FeedReader.ArticleTheme() {
+      // Local themes
+      string theme_dir = GLib.Environment.get_home_dir() + "/.feedreader/themes/";
```

Comment:
I'm pretty sure you can't use this path with flatpaks. Also there is already `$HOME/.local/feedreader`.
You can access that via `GLib.Environment.get_user_data_dir() + "/feedreader/"`